### ISSUE TITLE
Feature-Request: Direct Links and rel me

### ISSUE THREAD

**Issue by egooner** [JulianPrieber/littlelink-custom#208]
I started looking for this so that I could back reference my Mastodon account.

The Littlelink server by TechnoTim provides this but I wanted to be able to host multiple accounts within the single instance so this fits much better and the implementation is much easier to use going forward.

I notice that the links created do not have the required rel="me ..." for Mastodon and also that the links have a local link first.
It would be good to be able to have an option that sets the link as a verification link which then includes the rel="me ..." but also removes the internal link and goes direct to the page instead.

I have added rel="me ..." in the running container and stripped out the internal link from littlelink.blade.php in /htdocs/resources/views but think a better solution would be to include an option on the link creation that then allows this to be used or not used in the php logic.

For now I will run with my local bypass but hope that this is something you could include.

**Comment by JulianPrieber** [comment]
This is a feature we are already working on. This will be included in the next update, which will release early next week.

You can already test this new feature on our beta demo page here: https://3.littlelink-custom.com/

Alternatively, if you don't want to wait for the full release, you can install the beta release right now. More on this here: https://littlelink-custom.com/beta-tester

We plan on expanding mastodon integration in the future. You might also be interested in this: https://github.com/JulianPrieber/littlelink-custom/discussions/203

**Comment by egooner** [comment]
Just tried this out and I can see the rel me for Mastodon but the URL is
still using the internal link.

I have tested it with Mastodon and it needs the proper link to validate the
site.

Richard

**Comment by JulianPrieber** [comment]
@MagicLike

**Comment by MagicLike** [comment]
> Just tried this out and I can see the rel me for Mastodon but the URL is still using the internal link. I have tested it with Mastodon and it needs the proper link to validate the site. Richard

Actually I tried it today and it worked perfectely fine... Can you describe me what you did / what you linked?

**Comment by egooner** [comment]
Hmm - apologies - could be a timing issue as I tried it and it did not
seem to work - just tried it again and it is now working - it could be
that I just gave it too little time before reverting my BIO.

Thanks - Interesting though as I am wondering how this validation is
working given that the generated back link is

***@***.***

I presume that they must do an actual connect to the link because I
just tried adding in a dummy site eg
***@***.*** and it failed to verify.

Thanks for the update - I look forward to it going live next week -
looks like I found this at the right time.

Richard
